File: src/java/src/mil/navy/nrl/norm/enums/NormNackingMode.java
package mil.navy.nrl.norm.enums;

/**
 * Enumeration of the different types on NORM nacking modes. This enumeration
 * maps to the native implementation and therefore is tied to the specific
 * version of NORM.
 * 
 * @author Jason Rush
 */
public enum NormNackingMode {
  NORM_NACK_NONE,
  NORM_NACK_INFO_ONLY,
  NORM_NACK_NORMAL;
}

File: src/dotnet/src/Mil.Navy.Nrl.Norm/Enums/NormNackingMode.cs
namespace Mil.Navy.Nrl.Norm.Enums
{
    /// <summary>
    /// The available nacking modes.
    /// </summary>
    public enum NormNackingMode
    {
        /// <summary>
        /// Do not transmit any repair requests for the newly received object.
        /// </summary>
        NORM_NACK_NONE,
        /// <summary>
        /// Transmit repair requests for NORM_INFO content only as needed.
        /// </summary>
        NORM_NACK_INFO_ONLY,
        /// <summary>
        /// Transmit repair requests for entire object as needed.
        /// </summary>
        NORM_NACK_NORMAL
    }
}
